### ISSUE TITLE
Flutter App

### ISSUE THREAD

**Issue by zachbwh** [zachbwh/dailyzach#2]
**As a**
Cherished friend or loved one of Zach
**I would like to**
Have an App installed on my phone
**So I can**
Receive push notifications that Zach has added a new image, and view it

---

**As a**
Zach
**I would like to**
Have an App installed on my phone
**So I can**
Receive push notifications for selfie requests and reply with a selfie

**Comment by zachbwh** [comment]
There are two reasons why I need to make a React Native application instead of a PWA.
1. I have cherished friends and loved ones who have iPhones and would not be able to install a PWA
2. I would like to be able to use native camera APIs instead of webcam APIs so I can take high quality selfies for my cherished friends and loved ones

**Comment by zachbwh** [comment]
I've changed my mind about using React Native for this project, instead opting for flutter, for the following reasons:
1. Admittedly, CV Driven Development. My personal projects are a vehicle for me to expand my skillset and React Native is not sufficiently different to what I already know. Flutter & Dart and significantly different to what I've worked with professionally so they present the best opportunity to me for personal growth.
2. Flutter in my brief experience seems much more beginner friendly than React Native. React Native has great community projects like expo that solve common app dev use cases, but as soon as you want to do something outside of those solved problems, it's been very unclear to me how to progress. In contrast, flutter seems to be a much more centralised and cohesive project and community. This becomes most useful when implementing features that are slightly off the beaten track. Things like Auth0 documentation are much more targeted towards the application stack I'm already working with, so it becomes easier for me as a beginner mobile app developer to start executing more complex tasks I could easily do on web.
